### ISSUE TITLE
Extra tokens in input file

### ISSUE THREAD

**Issue by JueXiuHuang** [luyaojie/Text2Event#11]
Hi, I'm confused about the <extra_id_0> and <extra_id_1> in json files under the data/text2tree folder.

In Huggingface's example, these extra_ids represent to their replaced token, such as
`The <extra_id_0> walks in <extra_id_1> park` and `<extra_id_0> cute dog <extra_id_1> the <extra_id_2>`.

But in json file, most event label contains multiple <extra_id_0> and <extra_id_1>, such as
`"text": "He will blow a city off the earth in a minute if he can get the hold of the means to do it .", "event": "<extra_id_0> <extra_id_0> Attack blow <extra_id_0> Place earth <extra_id_1> <extra_id_1> <extra_id_1>"`

<extra_id_0> and <extra_id_1> also appears when there is no event label:
`"text": "I am shook over the aftermath .", "event": "<extra_id_0>  <extra_id_1>"`

so what are the <extra_id_0> and <extra_id_1> mean?

**Comment by luyaojie** [comment]
Hi,

In the T5 pre-training, <extra_id_0> and <extra_id_1> are used as mask symbol for mask language modeling.
And in our fine-tuning stage, we use them as the structure symbols to combine the event schema and spans from the text.
<extra_id_0>  is ‘（’  in our paper, and <extra_id_1>  is ')'.

<extra_id_0> <extra_id_0> Attack blow <extra_id_0> Place earth <extra_id_1> <extra_id_1> <extra_id_1>
-> 
((Attack blow (Place earth) )

In this way, '<extra_id_0> <extra_id_1>' represents "()", which means there is no event in the sentence.

**Comment by JueXiuHuang** [comment]
Thanks for your clear explanation!
